### PR TITLE
Estructura HTML y CSS challenge

### PR DESCRIPTION
**Pull request by CRISTIAN-CRUZ-PARRA**
Estructura HTML y CSS maquetación challenge

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                <li><a href="https://github.com/CRISTIAN-CRUZ-PARRA">My Github</a></li>
+                
+             </ul>
+        </div>
+    </main>
+    <footer>
+        <p>&copy; 2024 Cristian Cruz Parra</p>
+    </footer>
+</body>
+</html>
```
Comment (tictools) [inline index.html:41]: empty line

File: index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <title>My Notes Application</title>
+    <link rel="stylesheet" href="sanitize.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <h1>My Notes Application</h1>
+    </header>
+    <main>
+        <div class="form__container">
```
Comment (tictools) [inline index.html:15]: Apply semantic tag (article or section)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                <input type="text" id="title" name="title" placeholder="Enter note title" required>
+
+                <label for="content">Content:</label>
+                <input class="form-height" type="text" id="content" name="content" placeholder="" required>
+
+                
+                <button type="submit">Add Note</button>
+            </form>
+        </div>
+        <div class="resources_container">
```
Comment (tictools) [inline index.html]: Apply semantic tag

File: index.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    </header>
+    <main>
+        <div class="form__container">
+            <form>
+                <h2>Create a New Note</h2>
+                <label for="title">Title:</label>
+                <input type="text" id="title" name="title" placeholder="Enter note title" required>
+
+                <label for="content">Content:</label>
+                <input class="form-height" type="text" id="content" name="content" placeholder="" required>
```
Comment (tictools) [inline index.html]:
```suggestion
                <textarea class="form__text" type="text" id="content" name="content" placeholder="" required>
```

File: index.html [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+                <label for="content">Content:</label>
+                <textarea class="form-height" id="content" name="content" placeholder="" required></textarea>
+
+                
+                <button type="submit">Add Note</button>
+            </form>
+        </div>
+        <div class="resources_container">
+             <ul class="no-list-style">
```
Comment (tictools) [inline index.html]: Suggestion: wrap ul element within nav tag.

**Review by tictools — APPROVED** [review]

**Comment by CRISTIAN-CRUZ-PARRA** [comment]
Challenge 00_semantic-html-css COMPLETED

**Pull request merged**